torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3086,6 +3093,9 @@ def issue_guard(guard: ShapeGuard) -> None:
                 **self.counter,
                 "num_guards": len(exprs),
                 "free_symbols": sum(1 for v in symbol_to_source.values() if v),
+                # The keys are meaningless from an aggregate perspective, so
+                # don't include them.  Biggest first.
+                "symbol_guard_counts": sorted(self.symbol_guard_counter.values(), reverse=True),
```

Comment:
Won't this not tell us much, unless we also include the dynamo source that the "culprit" symbols map to?

Or are you just trying to use this for now to get a rough sense of the distribution

Reply:
It doesn't tell us much at all, but that's OK, I want to look at this data in aggregate over all jobs so even knowing the source in this context wouldn't be very meaningful.